### PR TITLE
Convert Docs for the Junit5

### PR DESCRIPTION
**Pull request by raynigon**
Convert inline example code to use codeinclude macro for the JUnit5 Testframework documentation.
See #1158 for more details.

### CODE REVIEW

File: modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package org.testcontainers.junit.jupiter;
```
Comment (raynigon) [inline modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java:1]: unfortunately i had to create a new test class, to have the code representing nearly the same logic as the previous documentation

**Comment by raynigon** [comment]
Difference in the docs, left (**old**) contains the inline code, right (**new**) was generated from the unit tests.
junit

File: modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @Container
+    private static final MySQLContainer MY_SQL_CONTAINER = new MySQLContainer();
+
+    // will be started before and stopped after each test method
+    @Container
+    private PostgreSQLContainer postgresqlContainer = new PostgreSQLContainer()
+        .withDatabaseName("foo")
+        .withUsername("foo")
+        .withPassword("secret");
+    @Test
```
Comment (rnorth) [inline modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/MixedLifecycleTests.java:31]: Let's just squeeze in a newline to create some visual separation

```suggestion

    @Test
```

**Comment by rnorth** [comment]
Merged - thanks @raynigon!

**Pull request merged**